lib/hello_trema.rb

```diff
@@ -1,6 +1,6 @@
 # Hello World!
 class HelloTrema < Trema::Controller
   def switch_ready(datapath_id)
-    info format('Hello %#x!', datapath_id)
+    logger.info format('Hello %#x!', datapath_id)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.